alfresco-connector/config/rest-clients.yaml

```diff
@@ -1,8 +1,7 @@
 RestClients:
-  #Preconfigured for the Docker Based Community Edition running on localhost port 8080
-  #Admin Url: https//localhost:8080/share
-  #Swagger openApi: https://api-explorer.alfresco.com/api-explorer/definitions/alfresco-core.json
-  Alfresco (Alfresco Content Services):
```

Comment:
breaking change?

Reply:
if someone really used it in his app, yes ... but I think nobody did